front-packages/akeneo-design-system/src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -80,11 +91,12 @@ type StepProps = Override<
      * The label of the step.
      */
     children?: ReactNode;
+    index?: number;
```

Comment:
In order to be explicit about the fact that this props is calculated
```suggestion
    /**
     * @private
     */
    index?: number;
```